start.sh

```diff
@@ -11,8 +11,17 @@ python /usr/app/update-display.py
 # Save out the current env to a file so cron job can use it
 export -p > /usr/app/env.sh
 
+# Set default values if these env vars are not set
+if [[ -z "${ALTERNATE_FREQUENCY}" ]]; then
+  Alternate="0"
+  [[ -z "${UPDATE_HOUR}" ]] && UpdateHour='9' || UpdateHour="${UPDATE_HOUR}"
+else
+  Alternate="*/${ALTERNATE_FREQUENCY}"
+  UpdateHour='*'
+fi
+
 # Add the job to the crontab using update_hour var, defaulting to 9 AM
-(echo "0 ${UPDATE_HOUR:-9} * * * /usr/app/run-update.sh") | crontab -
+(echo "${Alternate} ${UpdateHour} * * * /usr/app/run-update.sh") | crontab -
```

Comment:
For some reason `cron` job does not trigger on my device, although I see that it's set properly. I also tested a custom cron schedule to update every 60 minutes (`*/60 * * * * /usr/app/run-update.sh`) on the master code and that also did not run. Although the original code runs fine on daily trigger.

Reply:
Hm @chrisys are you able to test if this works on yours? I wonder if it has anything to do with the display being different